.buildkite/hooks/post-command

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+# Only run purge script on Windows agents, as `premerge` runs on a Linux box
```

Comment:
We don't appear to have a premerge step, and we don't run on linux agents.

Reply:
removed